easybuild/easyconfigs/m/matplotlib/matplotlib-1.5.1-foss-2016a-Python-2.7.11-freetype-2.6.3.eb

```diff
@@ -0,0 +1,44 @@
+easyblock = 'Bundle'
+
+name = 'matplotlib'
+version = '1.5.1'
+versionsuffix = '-Python-%(pyver)s-freetype-2.6.3'
```

Comment:
try to avoid hardcoding the freetype version in two places:

```python
freetype_ver = '2.6.3'
versionsuffix = '-Python-%(pyver)s-freetype-%s' % freetype_ver
...
dependencies = [
    ...
    ('freetype', freetype_ver),
    ...
]
```